test/journeys/lib/test-helpers/space-widget/messaging.js

```diff
@@ -103,26 +104,28 @@ export function flagMessage(testObject, messageToFlag) {
   moveMouse(testObject.browser, elements.lastActivityText);
   testObject.browser.waitUntil(() =>
     testObject.browser
-      .element(elements.lastActivity)
-      .element(elements.flagButton)
+      .element(`${elements.lastActivity} ${elements.flagButton}`)
       .isVisible(),
-  'flag button is not visible');
+  1500, 'flag button is not visible when hovering');
 
   testObject.browser
-    .element(elements.lastActivity)
-    .element(elements.flagButton)
+    .element(`${elements.lastActivity} ${elements.flagButton}`)
     .click();
 
+  // Flag has a pending class while it waits for server upload
+  testObject.browser.waitUntil(() => testObject.browser
+    .element(`${elements.lastActivity} ${elements.highlighted}${elements.pendingAction} ${elements.flagButton}`)
```

Comment:
Looks like there's a space missing between the 2nd and 3rd element variables.

Reply:
That is to be expected, we need the selector to be `.isHighlighted.pending` to get it to select properly since both of those classes are on the same element.